src/endpoints.js

```diff
@@ -13,7 +13,7 @@ const GRAPH_API_BASE_HTTP = 'https://api.thegraph.com'
 const GRAPH_API_BASE_WS = 'wss://api.thegraph.com'
 
 export const IPFS_ENDPOINT = isLocalOrUnknownNetwork(CHAIN_ID)
-  ? 'http://127.0.0.1:8080/ipfs'
```

Comment:
I might still keep this, since when you're on a local chain, our IPFS node isn't going to have your content.

Reply:
yeah, this file should not be uploaded i change it for testing